subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskUpToDateIntegrationTest.groovy

```diff
@@ -472,6 +472,11 @@ class TaskUpToDateIntegrationTest extends AbstractIntegrationSpec {
                 }))
             }
         """
+        executer.expectDocumentedDeprecationWarning(
+            "Creating a tarTree from a resource without a backing file has been deprecated. " +
```

Comment:
Should we say `TarFileTree` here? `tarTree()` is the name of the method that creates the tree, we shouldn't use that here I think.

Reply:
I changed the wording.